cmd/bundle/dependencies.go

```diff
@@ -0,0 +1,48 @@
+package bundle
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/databricks/cli/bundle/deploy/terraform"
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/libs/flags"
+	"github.com/spf13/cobra"
+)
+
+type Dependencies struct {
+	Terraform *terraform.TerraformMetadata `json:"terraform"`
+}
+
+func newDependenciesCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "dependencies",
+		Short: "Prints CLI dependencies required for the bundle namespace",
+		Args:  root.NoArgs,
+		// This command is currently intended for the Databricks VSCode extension only
+		Hidden: true,
```

Comment:
We can make this public as well.

Reply:
Nobody should need to use this. I prefer putting this under a `debug` subcommand, hide that, and then show everything under the `debug` command.

Reply:
Moved the command to `debug terraform`